### DISCUSSION TITLE
Safety and risk of fire

### DISCUSSION THREAD

**Discussion by Philippe734** [BitMaker-hub/NerdMiner_v2#233] [General]
Hello,
What about the safety and risk of fire if cooling suddenly stop while nobody at home ?
Is there a risk of fire ?
Does the system will halt if temperature goes up to 80°C?

**Comment by sany3001** [comment]
I am running 5 pieces without any cooling and without box and I am not worried about the fire in this case. (I am more worried from non-stop running fridge for example - especially when due to potential risk of planet overheating they installed immediate risk of fire in my household by putting flammable coolant into my fridge)

**Reply by Philippe734** [reply]
Without cooling, wahoo i'm surprise, thanks for the information. What is the temperature in that case ?

**Comment by sany3001** [comment]
Unfortunatelly I have no idea about temperature. There is open PR with measurement of temperature of the chip. I wanted to give it a try, but no prio for now. Still hoping it will be merged in mean time.
